instruqt/avoiding-installation-pitfalls/03-adding-preflights-to-the-chart/assignment.md

```diff
@@ -20,7 +20,7 @@ timelimit: 600
 ---
 
 There are many more preflight checks we could define for the
-Harbor registry, but let's stop here and shift to how we can
+Slackernews registry, but let's stop here and shift to how we can
```

Comment:
This next section, on first read, is a little bit confusing - we just ran preflight checks that checked a cluster, but now I'm learnig that prelight checks run _outside_ of a cluster, then there are bullet points that explain, the reasons we are going to make it a Secret are because we want to store it in a cluster

I think maybe talking about it in terms of the CRD angle might be more straightforward way of explaining why we want to wrap it in a Secret - like, as a Helm author, or a software builder, I think that makes more sense that maybe because we don't want to create a requirement that a cluster have a CRD for preflights, because we want to be able to use it before we install anything to the cluster, then we wrap it in a Secret so that preflight knows what to do with it, but Helm/Kubernetes don't need to know what a Preflight kind is.

Reply:
It, of course, gets worse, because we _do_ run outside of the cluster but probably shouldn't based on what users are trying to figure out. I reworked it to draw the same type of distinction without assuming everyone understands how CRDs work.